torchbenchmark/operators/flash_attention/operator.py

```diff
@@ -128,6 +128,7 @@ class Operator(BenchmarkOperator):
     def __init__(self, tb_args: argparse.Namespace, extra_args: Optional[List[str]] = None):
         super().__init__(tb_args, extra_args)
         args = parse_op_args(self.extra_args)
+        self.use_cuda_graphs = False
```

Comment:
I wonder why we need to turn off cuda_graphs.

Reply:
It is not about colfax_cutlass, but with ThunderKittens (https://github.com/pytorch/benchmark/pull/2370) mine fails with error:

```
Caught exception, terminating early with partial results
Traceback (most recent call last):
  File "/home/xz/git/benchmark/torchbenchmark/util/triton_op.py", line 558, in run
    y_vals: Dict[str, BenchmarkOperatorMetrics] = functools.reduce(
                                                  ^^^^^^^^^^^^^^^^^
  File "/home/xz/git/benchmark/torchbenchmark/util/triton_op.py", line 546, in _reduce_benchmarks
    acc[bm_name] = self._do_bench(
                   ^^^^^^^^^^^^^^^
  File "/home/xz/git/benchmark/torchbenchmark/util/triton_op.py", line 753, in _do_bench
    metrics.latency = triton.testing.do_bench_cudagraph(
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/xz/miniconda3/lib/python3.11/site-packages/triton/testing.py", line 46, in do_bench_cudagraph
    with torch.cuda.graph(g):
  File "/home/xz/miniconda3/lib/python3.11/site-packages/torch/cuda/graphs.py", line 186, in __exit__
    self.cuda_graph.capture_end()
  File "/home/xz/miniconda3/lib/python3.11/site-packages/torch/cuda/graphs.py", line 84, in capture_end
    super().capture_end()
RuntimeError: CUDA error: operation failed due to a previous error during capture
CUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.
For debugging consider passing CUDA_LAUNCH_BLOCKING=1
Compile with `TORCH_USE_CUDA_DSA` to enable device-side assertions.

```
So I am thinking cudagraph might not work with ThunderKittens?